src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -129,6 +129,12 @@
 
 public class ExchangeServiceTest extends VertxTest {
 
+    private static final String CUR2_CONVERSION_ERROR =
```

Comment:
i'd say it is not strongly desired for constants in unit-tests.
it is ok to duplicate messages.